submission_runner.py

```diff
@@ -312,10 +329,13 @@ def train_once(
                                                  hyperparameters)
     workload.attach_metrics_logger(metrics_logger)
 
-  global_start_time = time.time()
+  if USE_PYTORCH_DDP:
```

Comment:
Lines 332 and 333 are redundant now and can be removed, since `if USE_PYTORCH_DDP` evaluates to true, then `get_time = _get_time_ddp` which calls synchronize

Reply:
fixed, forgot to clean that up